grpc_stdio.go

```diff
@@ -101,10 +100,10 @@ func newGRPCStdioClient(
 	// Connect immediately to the endpoint
 	stdioClient, err := client.StreamStdio(ctx, &empty.Empty{})
 
-	// If we get an Unavailable error, this means that the plugin isn't
+	// If we get an Unavailable or Unimplemented error, this means that the plugin isn't
 	// updated and linking to the latest version of go-plugin that supports
 	// this. We fall back to the previous behavior of just not syncing anything.
-	if status.Code(err) == codes.Unavailable {
+	if status.Code(err) == codes.Unavailable || status.Code(err) == codes.Unimplemented {
```

Comment:
Still a bit confused of this check since an error was never returned from line 101 above in my case.